web/public/js/controllers/manage-dojo-users-controller.js

```diff
@@ -153,7 +156,7 @@ function cdManageDojoUsersCtrl($scope, $state, $q, cdDojoService, alertService,
   $scope.actions = {
     email: {
       ngShow: function () {
-        return $scope.selectedItems.length === 1;
+        return $scope.selectedItems.length === 1 && !$scope.emailPopup.isVisible;
```

Comment:
yes, to not make the emailing function confusing with both "to" emails from the popup and the user email at the bottom